Closes #161

trishalaneeraj opened the pull request:
Closes #161 

### Checkbox

- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `biodatasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_BIGBIO_VERSION` variables.

NOTE: _LICENSE is "Unknown" in this case

- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [X] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `BigBioConfig` for the source schema and one for a bigbio schema.
- [X] Confirm dataloader script works with `datasets.load_dataset` function.
- [X] Confirm that your dataloader script passes the test suite run with `python -m tests.test_bigbio biodatasets/my_dataset/my_dataset.py`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files. (NOT APPLICABLE)

debajyotidatta commented on biodatasets/mayosrs/mayosrs.py:
```diff
@@ -0,0 +1,190 @@
+                {
+                    "text_1": datasets.Value("string"),
+                    "text_2": datasets.Value("string"),
+                    "label": datasets.Value("string"),
```
Change to float in the source schema?

trishalaneeraj commented:
Hi @debajyotidatta I've addressed your comments in a new PR: https://github.com/bigscience-workshop/biomedical/pull/479

Could you please review the changes there and I'll go ahead and close this one?
